### ISSUE TITLE
could not update wallet: wallet need scan

### ISSUE THREAD

**Issue by copayedd** [bitpay/copay#9325]
I am receiving an error message on both my phone app and on computer.

could not update wallet: wallet need scan. wallet needs address scan.

I have tried the following...

1. I tried to clear the transaction history cache. that didn't help

2. I tried to scan addresses for funds. that didn't help.

3. Tried to delete the app and reinstall on my phone. When i tried restoring from back up seed i was prompted with the following...  "could not import: wallet need scan: wallet needs addresses scan.

4. Currently trying to use the copay recovery tool on my computer and it has been scanning for an hour and seems to be stuck on  scanned addresses: 45

Could someone please help? trying to withdraw my funds :(

Thank You

**Comment by copayedd** [comment]
After having a bit of a play around i have found out that this indeed is some type of bug. I created a new wallet on bitpay just to see if it was working fine. I was able to generate new addresses and everythig seemed to be working fine. Here is when the problems occurred.... As soon as i scanned addresses for funds or as soon as i cleared the cache i was prompted with the same error message i got from the beginning which is "could not update wallet: wallet need scan. wallet needs address scan." From here on in the wallet seems to be corrupted and will not go back to its original state of being able to generate listings or to refresh/update balance/transactions. From here on in, even if you have backed up your seed you will not be able to restore from back up. I tried and was prompted with the error message "could not import: wallet need scan: wallet needs addresses scan."

**Comment by PsychoSterope** [comment]
I am having the same issue after trying to top-up my BitPay visa, wallet just broke and funds are missing. Any ideas as to how to fix this? I am having the same issue on both mobile and desktop app.

**Comment by mytokondria** [comment]
Any reponse on how to resolve this??? I'm stucked here... Fund cannot be accessed and it's needed for urgent masters

**Comment by abhi0497** [comment]
Anyone found any solution?

**Comment by matiu** [comment]
Hi. can you try to scan addresses again? There were some delays on the
blockexplorer servers and we have upgrade them today.

Thanks.

**Comment by abhi0497** [comment]
Working fine now
Thank you!

**Comment by PsychoSterope** [comment]
Thanks for the suggestion, it did not work.. I think the system glitched out. I did get two emails for a repeat transaction. I loaded on the 19th and again on the 20th, but the email I got confirming the load that should have been for the 20th was just a clone of the load on the 19th. I am guessing the system took the BTC and didn't process correctly. I am just glad it wasn't a much higher amount.

**Comment by BGalliotti** [comment]
Hello @copayedd 
You are welcome.

We are sorry you have had this bad experience.
- Have you tried reinstalling the app? 

Kind regards.

**Comment by PsychoSterope** [comment]
@BGalliotti Yes, on two different PC and my phone. I am flustered, but I can earn that little amount back quickly.

**Comment by BGalliotti** [comment]
Hello @PsychoSterope 
Thanks for reaching out to us.

Let us some time for researching what could be going on here.
We would appreciate your patience.

Thanks in advance.

Kind regards.

**Comment by PsychoSterope** [comment]
@BGalliotti I am unable to fill out the request because this error did not generate a transaction ID or Invoice ID.

For now, I don't think there is much anyone can do. But thank you for trying.
